compiler/rustc_resolve/src/access_levels.rs

```diff
@@ -183,14 +169,6 @@ impl<'r, 'ast> Visitor<'ast> for AccessLevelsVisitor<'ast, 'r> {
                 visit::walk_item(self, item);
                 self.prev_level = orig_level;
             }
-
-            ast::ItemKind::ForeignMod(ForeignMod { ref items, .. }) => {
-                for nested in items {
-                    if nested.vis.kind.is_pub() {
-                        self.set_access_level(nested.id, access_level);
-                    }
-                }
-            }
```

Comment:
`fn set_exports_access_level` still doesn't visit items removed here and below, so they are no longer visited.

`fn set_exports_access_level` should either visit all items and not only imports (probably preferable long term), or this code needs to be restored.